### ISSUE TITLE
Allow to choose size of input group in Bootstrap 4

### ISSUE THREAD

**Issue by medihack** [django-crispy-forms/django-crispy-forms#1071]
* Package version: 1.9.2
* Django version: 3.1.2
* Python version: 3.8.5
* Template pack: (Optional)

### Description:

Currently, it's not possible to set the size of a Bootstrap Input group without overwriting the template.

According to the [Bootstrap documentation](https://getbootstrap.com/docs/4.5/components/input-group/#sizing) a class `input-group-sm` or `input-group-lg` must be added to the element that has the class `input-goup`.
Unfortunately, this is not possible as the [class there in the template is hardcoded](https://github.com/django-crispy-forms/django-crispy-forms/blob/2f1ad1fff035ef1906dfa012e42ac042465af22b/crispy_forms/templates/bootstrap4/layout/field_with_buttons.html#L11).

### Solution:

We could reuse the `input-size` field as it is in the [wrong place anyway](https://github.com/django-crispy-forms/django-crispy-forms/blob/2f1ad1fff035ef1906dfa012e42ac042465af22b/crispy_forms/templates/bootstrap4/layout/field_with_buttons.html#L13). At the same time [`PrependedAppendedText`](https://github.com/django-crispy-forms/django-crispy-forms/blob/6c0ffe2f417bdd6463d87905844ddb86b112cc96/crispy_forms/bootstrap.py#L23) could be fixed as `input-lg` and `input-sm` only seems to be available in Bootstrap 3 and not Bootstrap 4.

**Comment by medihack** [comment]
Just found out that an alternative solution to set the size is to simply change the classes of the elements itself (I see no difference to using `input-group-sm`):

```
FieldWithButtons(
  Field(field_name, css_class="custom-select custom-select-sm"),
  Submit("", "Set", css_class="btn-sm"), css_class="input-group-sm",
),
```

**Comment by smithdc1** [comment]
A pull request would be well received as it will help me understand what your issue is. :-)

Thanks

P.s. I expect tests to fail on Django master. It is on my list to fix.

**Comment by smithdc1** [comment]
Fixed in #1094 and #1159
